core/domain/search_services_test.py

```diff
@@ -273,6 +309,39 @@ def test_demo_explorations_are_added_to_search_index(self):
         results, _ = search_services.search_explorations('Welcome', 2)
         self.assertEqual(results, ['0'])
 
+    def test_update_public_exploration_status_in_search(self):
```

Comment:
Same question as above.

Reply:
Removed the function as it was not used anywhere